### PR TITLE
Remove ephemeral examples from CI

### PR DESCRIPTION
**Pull request by ritazh**
Signed-off-by: Rita Zhang <rita.z.zhang@gmail.com>

### CODE REVIEW

File: library/general/allowedrepos/samples/repo-must-be-openpolicyagent/example_disallowed_initcontainer.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
```
Comment (sozercan) [inline library/general/allowedrepos/samples/repo-must-be-openpolicyagent/example_disallowed_initcontainer.yaml:1]: do we need this still? does `both` scenario cover this too?

File: library/general/allowedrepos/suite.yaml [hunk 2]
```diff
@@ -12,24 +12,24 @@ tests:
...
-    object: samples/repo-must-be-openpolicyagent/disallowed_both.yaml
+    object: samples/repo-must-be-openpolicyagent/example_disallowed_both.yaml #disallowed_all.yaml
     assertions:
-    - violations: 3
+    - violations: 2 #3
     - message: initContainer
       violations: 1
     - message: container
       violations: 1
-    - message: ephemeralContainer
```
Comment (sozercan) [inline library/general/allowedrepos/suite.yaml:34]: do gator suite files break CI too?

File: library/general/allowedrepos/samples/repo-must-be-openpolicyagent/example_disallowed_initcontainer.yaml (hunk 1 above)
Comment (ritazh) [reply library/general/allowedrepos/samples/repo-must-be-openpolicyagent/example_disallowed_initcontainer.yaml:1]: We do need it. This test case has a positive case and a negative case where the initcontainer is not pulling allowed repo, whereas `both` has 2 violations for both init and container and `all` has 3 violations for init, container, and ephemeral.

File: library/general/allowedrepos/suite.yaml (hunk 2 above)
Comment (maxsmythe) [reply library/general/allowedrepos/suite.yaml]: gator tests should be well-behaved

File: library/general/disallowedtags/suite.yaml [hunk 3]
```diff
@@ -23,10 +23,10 @@ tests:
     object: samples/container-image-must-not-have-latest-tag/example_disallowed_tag.yaml
     assertions:
     - violations: yes
-  - name: single-disallowed-tag-ephemeral
-    object: samples/container-image-must-not-have-latest-tag/example_disallowed_tag_ephemeral.yaml
-    assertions:
-    - violations: yes
+  # - name: single-disallowed-tag-ephemeral
```
Comment (maxsmythe) [inline library/general/disallowedtags/suite.yaml:26]: Do we need to disable the suite tests? they should be unaffected by the issues live k8s tests have

**Comment by ritazh** [comment]
@sozercan @maxsmythe addressed your comments. PTAL

**Review by sozercan — APPROVED** [review]
Thank you! LGTM

**Pull request merged**